app/jobs/health/deliver_scheduled_documents_job.rb

```diff
@@ -0,0 +1,29 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Health
+  class DeliverScheduledDocumentsJob < BaseJob
+    queue_as ENV.fetch('DJ_LONG_QUEUE_NAME', :long_running)
+
+    def perform(user)
+      scheduled_document_scope.each do |scheduled_document|
+        next unless scheduled_document.check_hour
+        next unless scheduled_document.should_be_delivered?
+        next unless scheduled_document.deliver(user)
```

Comment:
Can we move this to a class method so we can only kick-off the job if it actually needs to run?  And check it in the rake task?